rskj-core/src/main/java/co/rsk/util/IpUtils.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.util;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by mario on 13/07/17.
+ */
+public class IpUtils {
+    private static final Logger logger = LoggerFactory.getLogger(IpUtils.class);
+
+    private static final String IPV6_INPUT_FORMAT = "^\\[(.*)\\]:([0-9]{1,})";
```

Comment:
* Square brackets are not mandatory for IPv6 addresses if they are not part of an URL or if a port number is not specified. But if there won't be a default port for nodes then it is correct.
* Second group of regex for port number could limit to 5 digits and allow IPs without port